build/charts/antrea/templates/agent/clusterrole.yaml

```diff
@@ -5,6 +5,18 @@ metadata:
   labels:
     app: antrea
 rules:
+  - apiGroups:
+      - stats.antrea.io
+    resources:
+      - networkpolicystats
+      - antreaclusternetworkpolicystats
+      - antreanetworkpolicystats
```

Comment:
There should be np reason to add these existing resources here.

Reply:
OK, I will remove them.